docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+│       └── client-tls-ca-cert.pem
+
+
+```
+
+There are four sections in the folder structure above to consider:
+- **Orderer organization MSP:** the `organizations/ordererOrganizations/ordererOrg1.example.com/msp` folder contains the orderer organization MSP that includes the `cacerts` and  `tlscacerts` folders that you need to create and then copy in the root certificates (`ca-cert.pem`) for the organization CA and TLS CA respectively. If you are using an intermediate CA, you also need to include the corresponding `intermediatecerts` and `tlsintermediatecerts` folders.
+- **Orderer local MSP:** the `organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/msp` folder, also known as the orderer local MSP, contains the enrollment certificate and private key for the ordering service `osn1` node. This folder is automatically generated when you enroll the orderer identity with the organization CA.
+- **TLS certificates:** the `organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls` folder contains the TLS certificate (`tls-cert.pem`) and private key (`tls-key.pem`) for the ordering service `osn1` node as well as the TLS CA root cert `tls-ca-cert.pem`. You will need to copy those files into this folder or point to where they exist on your file system.   
+- **Admin client certificates** - `admin-client/` folder contains the TLS certificates for the admin client that will be issuing the `osadmin` commands. The connection between the admin client that calls the `osnadmin` CLI and an orderer requires mutual TLS, although it is not required for the network itself. This means the admin client needs to enroll and register with a TLS CA to generate the TLS certificate (`client-tls-cert.pem`) and private key (`client-tls-key.pem`) that is provided to the `osnadmin` CLI. You will need to copy those certificates and the client TLS CA root cert (`client-tls-ca-cert.pem`) into this folder or point to where they exist on your file system. For simplicity of the tutorial, you could use the same TLS CA that the orderer organization uses. On a production network, they would likely be separate TLS CAs.  
+
+Certificate names used in this example are for illustration purposes and may not reflect the actual names of the certificates generated by the CA. When you generate the certificates you can rename them accordingly to make it easier to differentiate them.
+
+**Important:** You need to create the `config.yaml` file in the orderer organization MSP and orderer local MSP. This file enables "Node OU" support for the channel, a feature that allows the channel to recognize certificates that contain the "admin" OU as representing organization admins (who are admins of the ordering nodes owned by the organization by default). Learn more in the [Fabric CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#nodeous) documentation.
```

Comment:
"This file enables "Node OU" support for the channel"

I'm forgetting right now... is this Node OU support for both the organization msp on the channel, and for the local msp admin?
If so, what is the difference between what a local msp admin can do on an OSN (as authorized by the "admin" node OU), versus what a osnadmin client can do on an OSN (as authorized by the channel participation API mutual TLS)?

Reply:
Yes. The Node OU support is for both the organization msp on the channel, and for the local msp admin.
My understanding is that an ordering node admin (local MSP) really has no special privileges at all -- unlike a peer admin which is required to install a smart contract on a peer.

But we know that  osnadmin client can join an orderer to a channel and remove it from the channel and list the channels that the orderer is a member of.

@denyeart Not sure if I've addressed your comment here? Perhaps there is something more you need me to clarify in the doc?

Reply:
@pamandrejko I added clarifications to this paragraph in the orderer deployment guide:
https://github.com/hyperledger/fabric/pull/2080#discussion_r523103698

The discussion about osnadmin authorization versus orderer local msp "admin" was just an observations between us, that we kind of have two different representations of an orderer node admin now. I expect this will get resolved when we shift from TLS to a new authentication framework for APIs like these.

Reply:
Great. I saw that and dropped it into this tutorial as well:

**Important:** You need to create the `config.yaml` file and add it to the organization MSP and local MSP folder for each ordering node. This file enables Node OU support for the MSP, an important feature that allows the MSP's admin to be identified based on an "admin" OU in an identity's certificate. Learn more in the [Fabric CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#nodeous) documentation.